adapters/yssp/yssp.go

```diff
@@ -0,0 +1,213 @@
+package yssp
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+
+	"github.com/mxmCherry/openrtb/v15/openrtb2"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type YSSPAdapter struct {
```

Comment:
Please name this `adapter`. There is no need to repeat the package name. There is no need to export this type with an uppercase letter.